src/app.ts

```
@@ -91,7 +110,10 @@ app.use("/users", userRoutes);
 app.use("/merchants", merchantRoutes);
 app.use("/webhook-queue/merchant", merchantWebhookQueueRoutes);
 app.use("/reports/transactions", transactionReportsRoutes);
-//app.use("/api/v1/stellar", stellarContractRoutes);
+//app.use("/audit-log", auditLogRoute);
+app.use("/api/v1/stellar", stellarContractRoutes);
```

ESLint

[error] 'stellarContractRoutes' is not defined.

(no-undef)